src/executor.rs

```diff
@@ -141,6 +144,25 @@ fn invoke_dynamic(
         previous_syscall_handler
     });
 
+    // Order matters, for the libfunc impl
+    // https://github.com/starkware-libs/sequencer/blob/1b7252f8a30244d39614d7666aa113b81291808e/crates/blockifier/src/execution/entry_point_execution.rs#L208
+    let builtin_costs_array: &[u64] = &[
+        token_gas_cost(CostTokenType::Const) as u64,
+        token_gas_cost(CostTokenType::Pedersen) as u64,
+        token_gas_cost(CostTokenType::Bitwise) as u64,
+        token_gas_cost(CostTokenType::EcOp) as u64,
+        token_gas_cost(CostTokenType::Poseidon) as u64,
+        token_gas_cost(CostTokenType::AddMod) as u64,
+        token_gas_cost(CostTokenType::MulMod) as u64,
+    ];
```

Comment:
Since order matters and every field has a different meaning, maybe we could use a struct instead? Then they'll have names associated with the number.

Reply:
thats a good idea